tests/e2e/testfiles/values.yaml

```diff
@@ -13,6 +13,19 @@
     - "Configure authorization-proxy-server for [powerflex] for CR [1]"
     - "Delete custom resource [1]"
 
+- scenario: "Install Authorization CRDs"
+  paths:
+    - "testfiles/authorization-templates/csm_authorization_crds.yaml"
+  modules:
+    - "authorizationproxyserver"
+  steps:
```

Comment:
Should existing test scenarios incorporate CRD steps now?

Reply:
Yes, they should. I'm working on fixing auth proxy test now and will include CRD step in other scenarios where it is needed. Just wanted to leave this PR specifically for CRD e2e and not mix those changes here